AnalysisStep/interface/ExtendedBranch.h

```diff
@@ -121,7 +121,8 @@ namespace BranchHelpers{
     }
     void createBranch(varType* defVal_){
       if (!isVector && defVal_!=0){
-        if (defVal!=0) delete defVal; defVal = new varType;
+        if (defVal!=0) delete defVal;
```

Comment:
Isn't this equivalent?

Reply:
Yes it is.  The compiler in 94X was giving a warning about "confusing indentation" or something like that.  I wanted it to be quiet.